QUIZ APP/quiz.js

```diff
@@ -0,0 +1,228 @@
+document.addEventListener("DOMContentLoaded", function() {
+    
+    const questions = [
```

Comment:
It's better to move static (fixed) objects like this to a separate file to separate data from logic. Usually, in common practice, data like this is stored as a json file.